deploy/distrib/pom.xml

```diff
@@ -263,13 +280,45 @@
-                                <move todir="${jboss_folder}/modules/com/h2database/h2/main">
+                                <copy todir="${jboss_folder}/modules/com/h2database/h2/main">
                                     <fileset dir="${project.build.directory}/common-libs">
                                         <include name="h2-*" />
                                     </fileset>
-                                </move>
+                                </copy>
+
+                                <copy todir="${jboss_folder}/modules/org/bonitasoft/bonita-tomcat-valve/main">
+                                    <fileset dir="${project.build.directory}/common-libs">
+                                        <include name="bonita-tomcat-valve-*" />
+                                    </fileset>
+                                </copy>
+
+                                <!-- wildfly -->
+                                <!-- Copy all resources from project to wildfly -->
+                                <copy overwrite="true" todir="${wildfly_folder}">
+                                    <fileset dir="${project.basedir}/src/main/resources/wildfly" />
+                                </copy>
+                                <replace file="${wildfly_folder}/modules/system/layers/base/com/h2database/h2/main/module.xml" token="#h2.version#"
+                                         value="${h2.version}" />
+                                <replace file="${wildfly_folder}/standalone/deployments/h2.sar/META-INF/jboss-deployment-structure.xml" token="#jboss-h2-mbean.version#"
+                                         value="${jboss-h2-mbean.version}" />
+
+                                <!-- Copy ear-->
+                                <copy todir="${wildfly_folder}/standalone/deployments" file="${project.build.directory}/ear-with-server/bonita-all-in-one-${bonita.engine.version}.ear" />
+
+                                <!-- Copy the libs  -->
+                                <copy todir="${wildfly_folder}/standalone/deployments/h2.sar">
+                                    <fileset dir="${project.build.directory}/common-libs">
+                                        <include name="bonita-wildfly-h2-mbean-*" />
+                                    </fileset>
+                                </copy>
+                                <touch file="${wildfly_folder}/standalone/deployments/h2.sar.dodeploy" />
+                                <copy todir="${wildfly_folder}/modules/system/layers/base/com/h2database/h2/main">
+                                    <fileset dir="${project.build.directory}/common-libs">
+                                        <include name="h2-*" />
+                                    </fileset>
+                                </copy>
 
-                                <move todir="${jboss_folder}/modules/org/bonitasoft/bonita-tomcat-valve/main">
```

Comment:
no because it is located inside the `wildfly_folder` whereas the `bonita-wildfly-h2-mbean` and `h2` jars are not. That was why we add an issue.